src/index.d.ts

```diff
@@ -22,5 +22,10 @@ export function formatCurrency(
   isoCode: string,
   locale: string,
   raw: boolean,
-  noDecimal: boolean
+  noDecimal: boolean | noDecimalObject
```

Comment:
proposed improvement for clarity

rename this argument to `decimalConfig` ?

and below the object instead of dp and sf, we can name them in full such as
`decimal_places` and `significant_figures` ?

@rachelg98